### PR TITLE
implement basic snippet completions for vscode & codemirror

### PR DESCRIPTION
**Pull request by OskarDamkjaer**

### CODE REVIEW

File: packages/vscode-extension/package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "version": "2.0.0-next.4",
   "publisher": "Neo4j Inc.",
   "activationEvents": [
-    "onLanguage:cypher"
+    "onLanguage:javascript",
+    "onLanguage:typescript"
```
Comment (ncordon) [inline packages/vscode-extension/package.json:11]: I think this is not from this pr right? Looks more like you were trying how to provide embedded support for Cypher?

File: packages/language-support/src/autocompletion/completionCoreCompletions.ts [hunk 2]
```diff
@@ -282,6 +283,13 @@ function calculateNamespacePrefix(
   return namespacePrefix;
 }
 
+const baseTriggerCharacters = ['.', '(', ')', ','];
+export const snippetTriggerCharacters = ['<', '-'];
```
Comment (ncordon) [inline packages/language-support/src/autocompletion/completionCoreCompletions.ts:287]: Did you solve the problem with the `-` and `<`?

File: packages/vscode-extension/package.json (hunk 1 above)
Comment (OskarDamkjaer) [reply packages/vscode-extension/package.json:11]: yes that's right

File: packages/language-support/src/autocompletion/completionCoreCompletions.ts (hunk 2 above)
Comment (OskarDamkjaer) [reply packages/language-support/src/autocompletion/completionCoreCompletions.ts:287]: yep, will open a new PR with only the relevant changes

**Comment by OskarDamkjaer** [comment]
closing in favor of #190

**Pull request closed without merging**